app/[lang]/chapters/[slug]/[lesson]/page.tsx

```diff
@@ -108,6 +108,10 @@ export default function Page({ params }) {
     return (
       <Portal id="head">
         <title>{`${title} - Saving Satoshi`}</title>
+        <meta
```

Comment:
There's a `Head.tsx` in the app which already has a viewport defined although I couldn't find it being imported/used in any application code - is this not being used and should it be perhaps?